### DISCUSSION TITLE
Support for use of environment variables in parameters

### DISCUSSION THREAD

**Discussion by localheinz** [community/community#25508] [Actions]
<p>Similar to the requested support for environment variables in if conditions, It would be nice to be able to use environment variables in <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepswith" rel="nofollow noopener">jobs.&lt;job_id&gt;.steps.with</a> </p>
<p>This currently fails, see for example <a href="https://github.com/ergebnis/php-library-template/pull/203" rel="nofollow noopener">https://github.com/ergebnis/php-library-template/pull/203</a>:</p>
<pre><code>on:
  - pull_request
  - push

name: "Continuous Integration"

env:
  REQUIRED_PHP_EXTENSIONS: "mbstring"

jobs:
  coding-standards:
    name: "Coding Standards"

    runs-on: ubuntu-latest

    steps:
      - name: "Checkout"
        uses: actions/checkout@v2.0.0

      - name: "Install PHP with extensions"
        uses: shivammathur/setup-php@1.6.1
        with:
          coverage: none
          extensions: "$REQUIRED_PHP_EXTENSIONS"
          php-version: 7.2
</code></pre>

**Comment by dawidd6** [comment, marked as answer]
<p>Try: </p>
<pre><code>${{env.REQUIRED_PHP_EXTENSIONS}}
</code></pre>
